### PR TITLE
Alfreco content app first example stack

### PR DESCRIPTION
**Pull request by eromano**
Hi guys this project looks amazing. Can I try to collaborate on it with the Alfresco stacks? 
thanks

### CODE REVIEW

**Comment by eromano** [comment]
is still maintained this repo?

**Comment by marcosnils** [comment]
yes

File: alfresco/alfresco-content-app/stack.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: '3.1'
+services:
+    app:
+        build: '.'
```
Comment (marcosnils) [inline alfresco/alfresco-content-app/stack.yml]: This won't work because it's using a `build` instruction and there's nothing to build here.

**Comment by eromano** [comment]
now should be ok

**Pull request closed without merging**